README.md

```diff
@@ -451,10 +451,10 @@ Unfortunately it's not possible to verify that the type of children elements are
 
 ## Contributing
 
-The project uses a monorepo structure for the packages managed by [yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/). To get started with the project, run `yarn` to install the required dependencies for each package:
+The project uses a monorepo structure for the packages managed by [yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) and [lerna](https://lerna.js.org). To get started with the project, run `lerna bootstrap` to install the required dependencies for each package:
 
 ```sh
-yarn
+lerna bootstrap
```

Comment:
AFAIK with `"useWorkspaces": true` you don't have to do `lerna bootstrap`, `yarn install` will do just fine